mesonbuild/modules/unstable_external_project.py

```diff
@@ -55,7 +57,8 @@ def __init__(self,
                  configure_options: T.List[str],
                  cross_configure_options: T.List[str],
                  env: build.EnvironmentVariables,
-                 verbose: bool):
+                 verbose: bool,
+                 extra_depends: T.List[T.Union[BuildTarget, CustomTarget]]):
```

Comment:
typing-only imports need to either be quoted at the time of use or use future annotations.

Reply:
ah right, done.